adserver/models.py

```diff
@@ -123,6 +131,12 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse("publisher_report", kwargs={"publisher_slug": self.slug})
 
+    @property
+    def keywords(self):
```

Comment:
This function (or the model validator) needs to be fleshed out a bit more. For example, this is what .org does: https://github.com/readthedocs/readthedocs.org/blob/505be8cc8fabc0dbdf290336dd263c887ede459b/readthedocs/projects/tag_utils.py#L12-L29

- Lowercase the tags
- Slugify the tags
- Replace underscores
- `from taggit.utils import _parse_tags` also does some smart things like detecting that the user used spaces instead of commas. That's less required but nice.

Reply:
I've done some basic cleanup here, but we're not exposing this to users quite yet, so I think its fine to be a bit rough.